package on2020_02.on2020_02_05_Codeforces_Round__617__Div__3_.A___Array_with_Odd_Sum;

import net.egork.chelper.tester.NewTester;

import org.junit.Assert;
import org.junit.Test;

public class Main {
	@Test
	public void test() throws Exception {
		if (!NewTester.test("lib/test/on2020_02/on2020_02_05_Codeforces_Round__617__Div__3_/A___Array_with_Odd_Sum/A - Array with Odd Sum.json"))
			Assert.fail();
	}
}
